docs/doc/14-sql-commands/20-query-syntax/values.md

```diff
@@ -0,0 +1,74 @@
+---
+title: VALUES
+---
+import FunctionDescription from '@site/src/components/FunctionDescription';
+
+<FunctionDescription description="Introduced or updated: v1.2.65"/>
+
+The VALUES clause is used to define a set of rows explicitly for use in queries. It allows you to provide a list of values that can be used as a temporary table in your SQL statements.
+
+## Syntax
+
+```sql
+VALUES (value_1_1, value_1_2, ...), (value_2_1, value_2_2, ...), ...
+```
+- The VALUES clause is followed by sets of values enclosed in parentheses.
+- Each set of values represents a row to be inserted into the temporary table.
+- Within each set of values, the individual values are comma-separated and correspond to the columns of the temporary table.
+- Databend automatically assigns default column names like *col0*, *col1*, *col2*, and so on when you insert multiple rows without specifying column names.
+
+## Examples
+
+These examples demonstrate using the VALUES clause to show city data in various formats: directly, ordered by population, and with structured column aliases:
+
+```sql
+VALUES ('Toronto', 2731571), ('Vancouver', 631486), ('Montreal', 1704694);
+
+col0     |col1   |
+---------+-------+
+Toronto  |2731571|
+Vancouver| 631486|
+Montreal |1704694|
+
+VALUES ('Toronto', 2731571), ('Vancouver', 631486), ('Montreal', 1704694) ORDER BY col1;
+
+col0     |col1   |
+---------+-------+
+Vancouver| 631486|
+Montreal |1704694|
+Toronto  |2731571|
+
+SELECT * FROM (
+    VALUES ('Toronto', 2731571), 
+           ('Vancouver', 631486), 
+           ('Montreal', 1704694)
+) AS CityPopulation(City, Population);
+
+city     |population|
+---------+----------+
+Toronto  |   2731571|
+Vancouver|    631486|
+Montreal |   1704694|
+```
+
+These examples demonstrate how the VALUES clause can be used in a SELECT statement:
```

Comment:
When the VALUES clause is used in a SELECT statement, user can specify optional table and column names.
for example
```sql
SELECT * FROM (VALUES(1,1),(2,null),(null,5)) AS t(a,b)
```